util/test/send_email.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
```

Comment:
Nothing here looked off to me, but I'm not familiar with sending mail from python. I can spend more time looking at this if you want me to.